### PR TITLE
Group 2 word is solved 

### LINKED ISSUE #100: Add word_is_solved(word, guesses)
Group 2

### PR DESCRIPTION
**Pull request by jenettewu**
Resolves #100

### CODE REVIEW

File: hangman.py [hunk 1]
```diff
@@ -48,6 +48,8 @@ def word_is_solved(word, guesses):
     """
     pass
```
Comment (MrGallo) [inline hangman.py]: Need to remove `pass`.

**Review by MrGallo — CHANGES_REQUESTED** [review]
What issue is this closing?

File: hangman.py (hunk 1 above)
Comment (jenettewu) [reply hangman.py]: I’m adding the function word_is_solved to the gr.10 hangman code

Comment (MrGallo) [reply hangman.py]: `pass` is a placeholder than needs to be removed. Remove it an the code is good.

**Review by MrGallo — APPROVED** [review]

**Pull request merged**